connexion/apis/abstract.py

```diff
@@ -153,7 +142,26 @@ def __init__(self, specification, base_url=None, arguments=None,
         self.add_paths()
 
         if auth_all_paths:
-            self.add_auth_on_not_found()
+            self.add_auth_on_not_found(self.security, self.security_definitions)
```

Comment:
What do you think about in the original method to add `*args, **kwargs` just to signal that it might have arguments in the subclasses?